.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
I saw `.vscode` is in your `.gitignore` but the files are still here, to remove them, you can remove it from the ignored files, push a commit  to delete it then add it to the ignored files.

Reply:
@sirghiny I managed to get rid of the `.vscode` directory.